tests/fixtures/requirements-locked.txt

```diff
@@ -11,7 +11,8 @@ amqp==5.0.9 --hash=sha256:77fd4e1249d8c9923de34907236b747ced06e5467ecac1a7bb7115
     --hash=sha256:8c2f9abd47a9e8df7f0c3f091ce9497d011dc3b31effcf4c85a6e2b50f4114ef
     # via kombu
 
-attrs==20.2.0   # This is an inline comment
+# Whitespace is allowed in requirement specifiers
+attrs == 20.2.0   # This is an inline comment
```

Comment:
Is it possible to have both variants in the fixture?

Reply:
Yes. Both variants are already in this fixture. For example `alembic==1.10.3` is on line 8.